### PR TITLE
Blog post announcing instructor training materials

### PR DESCRIPTION
**Pull request by gvwilson**
@karinlag @apawlik can you please also review?

### CODE REVIEW

File: blog/2015/12/announcing-instructor-training-materials.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+root: ../../..
+author: "Greg Wilson"
+title: "Announcing Instructor Training Materials"
+date: 2015-12-04
+time: "02:00:00"
+category: ["Instructor Training", "Lessons"]
+---
+<!-- start excerpt -->
+<p>
+  We are pleased to announce that the lessons from our instructor training course
```
Comment (karinlag) [inline blog/2015/12/announcing-instructor-training-materials.html]: lessons for our instructor training

**Comment by karinlag** [comment]
Looks good.

:+1:

**Comment by jduckles** [comment]
Is this ready to merge @gvwilson?

**Comment by gvwilson** [comment]
I've got one small correction to make, then I'll do it.

**Pull request merged**